rules/scripts/gdb_test_coordinator.py

```diff
@@ -205,8 +205,14 @@ def gdb_maybe_consume_expected_line(line: str):
     # with the JTAG tap because opentitantool will reconfigure some GPIOs and
     # this will interfere with the JTAG tap (#17729)
     background = BackgroundProcessGroup()
-    background.run(console_command, "CONSOLE", COLOR_RED)
-    time.sleep(1)
+    console = background.run(console_command, "CONSOLE", COLOR_RED)
+    # Wait until the console is ready to display message (at this point we know
+    # it won't touch the GPIOs anymore)
+    console_ready = background.block_until_line_contains(
```

Comment:
Nit: Could you move the comment that starts with "Run OpenOCD, GDB, and the OpenTitanTool console in the background." above line 208?

Reply:
Will do.